spaceship.zsh

```diff
@@ -738,6 +746,25 @@ spaceship_julia() {
     "$SPACESHIP_JULIA_SUFFIX"
 }
 
+# JAVA
+# Show current Java version
+spaceship_java() {
```

Comment:
Is showing Java version reasonable? In case with Node.js or Ruby, we often change versions of runtime. is that the same for Java?